server/rw.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+// Data
+
+const fs = require('fs');
+
+// Write
+
+function objectsToCsv(objects) {
+    let letters = objects.map(function(num) {
+        return num.name.match(/[A-Z]/)[0];
+    });
+    let numbers = objects.map(function(num) {
+        return num.name.match(/\d/)[0];
+    });
+    let lastLetter = letters.sort().pop();
+    let lastNumber = numbers.sort().pop();
+    let file = [];
+    for (var i = 0; i < lastNumber; i++) {
+        let row = [];
+        for (var y = 'A'.charCodeAt(0); y < lastLetter.charCodeAt(0) + 1; y++) {
```

Comment:
do it before loop and assign to variables
'A'.charCodeAt(0)
lastLetter.charCodeAt(0)

Reply:
done